prow/plugins/approve/approve.go

```diff
@@ -236,15 +241,18 @@ func handleReview(log *logrus.Entry, ghc githubClient, oc ownersClient, githubCo
 	// genericCommentEventHandler handle this event. Approval commands override
 	// review state.
 	if isApprovalCommand(botName, opts.LgtmActsAsApprove, &comment{Body: re.Review.Body, Author: re.Review.User.Login}) {
+		log.Debug("Review constitutes approval, skipping event.")
```

Comment:
nit: This is a little confusing without the context of the comment.
/shrug

Reply:
Eh, we get the line number in the message, figured it would be fine